hack/README

```diff
@@ -0,0 +1,13 @@
+This README documents how to generate CRD yaml files using ./hack/update-crd.sh
+after updating /pkg/apis/volumesnapshot/v1beta1/types.go
+
+step 1: run ./hack/update-crd.sh from top directory, new yaml files should have
+        been created under ./config/crd
+step 2: add the github PR annotation to any yaml files updated into metadata
+        section of the generated type, i.e.,
+				api-approved.kubernetes.io: "https://github.com/kubernetes-csi/external-snapshotter/pull/139"
```

Comment:
Can we have multiple PR annotations like this?  For example, when we update it next time, should we still keep this PR 139 and add a new PR annotation on top of this?

Reply:
no we should not, I have updated the . README to indicate that